client/cmd/dexc/ui/config.go

```diff
@@ -19,12 +19,14 @@ import (
 )
 
 const (
-	maxLogRolls     = 16
-	defaultRPCAddr  = "localhost:5757"
-	defaultWebAddr  = "localhost:5758"
-	configFilename  = "dexc_mainnet.conf"
-	certsFilename   = "certs.json"
-	defaultLogLevel = "info"
+	maxLogRolls        = 16
+	defaultRPCAddr     = "localhost:5757"
+	defaultRPCCertFile = "dexc_rpc.cert"
+	defaultRPCKeyFile  = "dexc_rpc.key"
+	defaultWebAddr     = "localhost:5758"
+	configFilename     = "dexc_mainnet.conf"
```

Comment:
The other dcr progs use "dexc.conf", so dexc.conf is more appropriate here.  We could change that up given good reason, but I didn't see logic to change the file name for testnet e.g.  Sorry if I missed it though.